jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -304,25 +336,11 @@ pipeline {
                                         architecture: 'arm64',
                                         distribution: 'rpm'
                                     )
-
-                                    String buildManifestUrl = buildManifestObj.getUrl(JOB_NAME, BUILD_NUMBER)
-                                    String artifactUrl = buildManifestObj.getArtifactUrl(JOB_NAME, BUILD_NUMBER)
-                                    env.ARTIFACT_URL_ARM64_RPM = artifactUrl
-                                    echo "buildManifestUrl (arm64, rpm): ${buildManifestUrl}"
-                                    echo "artifactUrl (arm64, rpm): ${artifactUrl}"
-
```

Comment:
same here

Reply:
These are not used tho and we can add back later.

Reply:
Added now.